### PR TITLE
Forward DNS Server Group implementation

### PR DESCRIPTION
**Pull request by venkat-iblox**

### CODE REVIEW

File: internal/service/dns_config/model_config_forward_nsg.go [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+	"internal_forwarders": schema.ListAttribute{
+		ElementType:         types.StringType,
+		Optional:            true,
+		MarkdownDescription: "The resource identifier.",
+	},
+	"name": schema.StringAttribute{
+		Required: true,
+		PlanModifiers: []planmodifier.String{
+			stringplanmodifier.RequiresReplaceIfConfigured(),
+		},
```
Comment (mathewab) [inline internal/service/dns_config/model_config_forward_nsg.go:83]: is this required for `name`

Comment (venkat-iblox) [reply internal/service/dns_config/model_config_forward_nsg.go:83]: Not required. Wil remove it

File: internal/service/dns_config/api_auth_nsg_resource.go
[the remark this answers is not in the document]
Comment (venkat-iblox) [reply internal/service/dns_config/api_auth_nsg_resource.go:1]: Addressed.

File: internal/service/dns_config/api_view_resource.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ func (r *ViewResource) Metadata(ctx context.Context, req resource.MetadataReques
 
 func (r *ViewResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
 	resp.Schema = schema.Schema{
-		MarkdownDescription: ``,
+		MarkdownDescription: `Use this method to create a View object. Named collection of DNS View settings.`,
```
[the remark this answers is not in the document]
Comment (venkat-iblox) [reply internal/service/dns_config/api_view_resource.go]: Addressed.

File: internal/service/dns_config/api_forward_nsg_resource_test.go [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+	var resourceName = "bloxone_dns_forward_nsg.test_comment"
+	var v dns_config.ConfigForwardNSG
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccForwardNsgComment("terraform-acc-test.com", "This Forward zone is created through Terraform"),
```
[the remark this answers is not in the document]
Comment (venkat-iblox) [reply internal/service/dns_config/api_forward_nsg_resource_test.go]: Addressed :).

**Review by mathewab — APPROVED** [review]

**Pull request merged**